repos/exawind/packages/amr-wind/package.py

```diff
@@ -25,6 +25,9 @@ class AmrWind(bAmrWind):
     depends_on('hdf5+mpi', when='+hdf5+mpi')
     depends_on('h5z-zfp', when='+hdf5')
     depends_on('zfp', when='+hdf5')
+    depends_on("ninja", type="build")
+
+    generator = "Ninja"
```

Comment:
```suggestion
    @property
    def generator():
          if '+ninja' in self.spec:
              return "Ninja"
          else:
              return "Unix Makefiles"
```
does this work to make ninja optional

Reply:
Oh cool, yeah let me try that.